src/interfaces/special-resolution-interfaces/person-interface.ts

```diff
@@ -0,0 +1,12 @@
+export interface PersonIF {
+  givenName: string
+  additionalName?: string
+  familyName: string
+  email?: string
+}
+
+export const EmptyPerson: PersonIF = {
+  givenName: '',
+  familyName: '',
+  additionalName: ''
```

Comment:
The empty values should be the same as those returned by the API when a person object is fetched.

For example, will all of theses be empty strings? Or could they be null?

Here's what I'm trying to avoid for you: https://github.com/bcgov/business-edit-ui/blob/e7f3877a3f870a0ddba63c9524f8f97073614829/src/store/getters/state-getters.ts#L728

Reply:
Maybe the optional fields should be null?

Or maybe additionalName shouldn't be optional?